### DISCUSSION TITLE
My fastlogin cannot connect to the database on velocity

### DISCUSSION THREAD

**Discussion by LovelyGuYiMeng** [games647/FastLogin#1099] [Q&A]
Using the latest velocity version and the latest fastlogin version
I've made sure my database is configured correctly

The first picture appears when I use mysql
The second picture appears when I use mariadb
![QQ图片20231007030352](https://github.com/games647/FastLogin/assets/76251800/3d4b5bd5-4309-4d4d-ac84-ddba1c50fd29)
![QQ图片20231007030358](https://github.com/games647/FastLogin/assets/76251800/8d91cbac-c91e-44ca-bc74-f31ffe70af6f)

**Comment by TuxCoding** [comment]
Thanks. I'll have to change that. It seems to be necessarily because the Mariadb driver is no longer compatible with MySQL. 

Can I ask you which database software of these two you use?

**Reply by LovelyGuYiMeng** [reply]
I am using mariadb
